ext/oci8/oci8.stub.php

```diff
@@ -3,24 +3,24 @@
 /** @generate-function-entries */
 
 /** @param resource $statement */
-function oci_define_by_name($statement, string $column, mixed &$result, int $type = 0): bool {}
+function oci_define_by_name($statement, string $column, mixed &$var, int $type = 0): bool {}
 
 /**
  * @param resource $statement
  * @alias oci_define_by_name
  * @deprecated
  */
-function ocidefinebyname($statement, string $column, mixed &$result, int $type = 0): bool {}
+function ocidefinebyname($statement, string $column, mixed &$var, int $type = 0): bool {}
 
 /** @param resource $statement */
-function oci_bind_by_name($statement, string $param, mixed &$var, int $max_length = -1, int $type = 0): bool {}
+function oci_bind_by_name($statement, string $param, mixed &$var, int $length = -1, int $type = 0): bool {}
```

Comment:
I would retain $max_length in oci_bind_by_name (the same thing in PDO is called $maxLength). The comment was just about the lob_write, where the meaning is the same as for `fwrite()`.